### PR TITLE
MSPSDS-217 Use GOVUK elasticsearch instead of AWS

### PR DESCRIPTION
**Pull request by AnthonyPengelly**

### CODE REVIEW

File: config/elasticsearch.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+      :timeout: 5
+
+development:
+  <<: *default
+
+test:
+  <<: *default
+
+production:
+  :url: <%= ENV['VCAP_SERVICES'] && JSON.parse(ENV['VCAP_SERVICES'])["elasticsearch"][0]["credentials"]["uri"] %>
```
Comment (broder) [inline config/elasticsearch.yml]: Ew - would it be worth sending an email to PaaS support to ask if there's another way? Feels like our experience might be useful to them (whether or not there's a better way of doing this).

File: docs/Deploy from scratch.md [hunk 2]
```diff
@@ -35,8 +35,10 @@ Create an S3 bucket named `int-mspsds`. This bucket needs public _read_ access.
 
 ## Elastic Search
 
-Create an AWS Elasticsearch domain called `int-mspsds`, choosing a suitable instance size.
-For access, choose public, and then allow your int-mspsds user to access the resource (by pasting their ARN).
+Create an empty elasticsearch instance in the `int` space
+
+    cf marketplace -s elasticsearch
+    cf create-service elasticsearch small-ha-6.x mspsds-elasticsearch
```
Comment (broder) [inline docs/Deploy from scratch.md]: Is there a non-HA option? I feel like our service probably doesn't need high availability for now.

Comment (AnthonyPengelly) [reply docs/Deploy from scratch.md]: No, this is the only option (or small-ha-5.x). It is on the free tier though

File: config/elasticsearch.yml (hunk 1 above)
Comment (AnthonyPengelly) [reply config/elasticsearch.yml]: Will do!

**Pull request merged**